app/web/src/components/AttributesPanel/AttributesPanelItem.vue

```diff
@@ -474,7 +474,7 @@
     <div
       v-if="showValidationDetails && validation"
       :style="{ marginLeft: indentPx }"
-      class="text-red-400 flex flex-col bg-black pl-3 border-y border-red-400 pb-1 my-1"
+      class="flex flex-col pl-3 border pb-1 m-2 text-destructive-500 border-destructive-500 bg-destructive-100 dark:text-destructive-900 dark:border-destructive-900 dark:bg-destructive-500"
```

Comment:
This is likely not the final turn here, but what was there was definitely incorrect.